sklearn/decomposition/_pca.py

```diff
@@ -36,13 +37,16 @@ def _assess_dimension_(spectrum, rank, n_samples, n_features):
     Parameters
     ----------
     spectrum : array of shape (n)
-        Data spectrum.
+        Data spectrum (of type float).
     rank : int
         Tested rank value.
     n_samples : int
         Number of samples.
     n_features : int
         Number of features.
+    spectrum_cutoff : None
```

Comment:
I'd prefer `threshold` rather than `cutoff` particularly when the meaning is like "you have to be at least this tall to get through"

Reply:
yup, alright